apps/builder/package.json

```diff
@@ -56,6 +56,7 @@
     "@webstudio-is/trpc-interface": "workspace:^",
     "colord": "^2.9.3",
     "css-tree": "^2.3.1",
+    "css-box-shadow": "^1.0.0-3",
```

Comment:
The repo is archived
https://github.com/jxnblk/css-box-shadow

Reply:
You can use css-tree to get detailed ast and use it to get values.

Reply:
I tried the same before as the background parser. It became little tricky with all the edge cases. As unlike `background` we don't have a set of gradients to filter out the css-values. And yeah, i didn't use `css-tree` much before. So, i am going to change the logic to use it once again then 👍 
https://github.com/webstudio-is/webstudio-builder/blob/main/apps/builder/app/builder/features/style-panel/sections/backgrounds/background-parser.ts#L53

Yup, the repo is archived. Can it be forked and maintained or something ? Because regex approach for box-shadows looks much promising.

Reply:
@TrySound any suggestions on the fork approach 😄

Reply:
I'd say better css-tree since we already using it. It also does more precise parsing than regex.

Reply:
cool, css-tree it is 👍